demos/common/python/openvino/model_zoo/model_api/adapters/onnx_runtime_adapter.py

```diff
@@ -0,0 +1,79 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+try:
+    import onnxruntime as ort
+    onnxruntime_absent = False
+except ImportError:
+    onnxruntime_absent = True
+
+from .model_adapter import ModelAdapter, Metadata
+
+
+class ONNXRuntimeAdapter(ModelAdapter):
+    precisions = {
+        'float': 'FP32',
+        'int64': 'I32'
```

Comment:
not sure about the last one, there is difference between int64 and int32 for onnxruntime (please check nlp models)